src/main/java/com/amihaiemil/docker/Execs.java

```diff
@@ -34,6 +37,15 @@
  */
 public interface Execs {
 
+    /**
+     * Create a exec with ID or name of container.
+     * @param id ID or name of container.
+     * @param config Exec configuration.
```

Comment:
@lumimul Please also add the tag:

``@see <a href="https://docs.docker.com/engine/api/v1.40/#operation/ContainerExec">Create Exec</a>``

So the users know where to look for the Json Config format.

Reply:
fix it